pkg/entity/batch_job/batch_job.go

```diff
@@ -59,29 +67,53 @@ func del(id *string, cmd *cobra.Command) {
 
 func create(cmd *cobra.Command) {
 	flags := cmd.Flags()
-	batchJobFile := util.GetStringAndErr(flags, batchJobsFileFlagName)
+	batchJobFile := util.GetStringAndErr(flags, batchJobFileFlagName)
+	batchJobType := getBatchJobType(flags)
 
-	batchJobData, err := ioutil.ReadFile(batchJobFile)
+	batchJobData, err := os.ReadFile(batchJobFile)
 	if err != nil {
 		common.CliExit(err)
 	}
 
+	var batchJobWrapper *entities.BatchJobWrapper
+	if batchJobType == encryptionType {
+		batchJobWrapper = createEncryptionBatchJob(cmd, batchJobData, flags)
+	} else if batchJobType == microAggregationType {
+		batchJobWrapper = createMicroAggregationBatchJob(cmd, batchJobData)
+	}
+	createBatchJobRequest := &batch_jobs.CreateBatchJobRequest{Job: batchJobWrapper}
```

Comment:
In case of an encryption batchJob, shouldn't we also still fill out the batchjob field for backwards compatibility with the batch job agent?